### PR TITLE
[Git-Hub project]: realized Calculator class

### PR DESCRIPTION
**Pull request by Sofasmile**
- Created Calculator class
- Created `add()` method

### CODE REVIEW

File: src/main/java/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Calculator {
+    private int add(int a, int b) {
+        return a + b;
+    }
+}
```
Comment (DevIgork) [inline src/main/java/Calculator.java:5]:
```suggestion
}

```
it is a good practice to leave a blank line at the end of the class

**Review by HovorukhaBohdan — APPROVED** [review]
LGTM

File: src/main/java/Calculator.java (hunk 1 above)
Comment (forzen123212) [inline src/main/java/Calculator.java]: You need to add other methods

Comment (DPotapov93) [inline src/main/java/Calculator.java:5]: empty line

File: src/main/java/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class Calculator {
+    private int add(int a, int b) {
```
Comment (dima-semeniuk) [inline src/main/java/Calculator.java]: Method should be public

**Review by forzen123212 — CHANGES_REQUESTED** [review]
Good job!

File: src/main/java/Calculator.java (hunk 2 above)
Comment (fedorovychh) [inline src/main/java/Calculator.java]: good job, but make this method static please

File: src/main/java/Calculator.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+public class Calculator {
```
Comment (annton-shakur) [inline src/main/java/Calculator.java:1]: Goog job, though you may need to create an interface

**Review by DPotapov93 — CHANGES_REQUESTED** [review]

File: src/main/java/Calculator.java (hunk 2 above)
Comment (fedorovychh) [reply src/main/java/Calculator.java]: and please, do it public

**Review by DevIgork — CHANGES_REQUESTED** [review]

**Review by fedorovychh — CHANGES_REQUESTED** [review]

**Review by annton-shakur — CHANGES_REQUESTED** [review]

**Review by HovorukhaBohdan — APPROVED** [review]
👍

**Review by fedorovychh — APPROVED** [review]
good job

**Review by RostyslavKuzmych — APPROVED** [review]
Great job, but be attentive.

**Pull request merged**